projects/gui/src/newtournamentdialog.cpp

```diff
@@ -240,6 +254,35 @@ Tournament* NewTournamentDialog::createTournament(GameManager* gameManager) cons
 	t->setGamesPerEncounter(ui->m_gamesPerEncounterSpin->value());
 	t->setRoundMultiplier(ui->m_roundsSpin->value());
 
+	const QString& fileName = ui->m_suiteFileEdit->text();
```

Comment:
The const reference doesn't really make sense here. `ui->m_suiteFileEdit->text()` returns a `QString` object so this line should be:

`QString fileName = ui->m_suiteFileEdit->text();`